README.md

```diff
@@ -5,7 +5,9 @@ This repository contains the Modelica Language Specification, hosted at https://
 
 ## Description
 
-Modelica® https://modelica.org is a non-proprietary, object-oriented, equation based language to conveniently model complex physical systems containing, e.g., mechanical, electrical, electronic, magnetic, hydraulic, thermal, control, electric power or process-oriented subcomponents.
+Modelica is a language for modeling of cyber-physical systems, designed to support effective library development and model exchange.
+It is a modern language built on acausal modeling with mathematical equations and object-oriented constructs to facilitate reuse of modeling knowledge.
```

Comment:
Just marking so that we remember to update this when an agreement about the corresponding lines in chapters/introduction.tex is reached. For example, as suggested in [this comment](https://github.com/modelica/ModelicaSpecification/pull/3037/files/2ebc51e3a9d6b7957ad75f7bf4d641d54a0f5872..3f129e1322429b522f5e30b27136a17386bbc5e5#r753865316).

Reply:
Done.  OK?

Reply:
Good. Resolving.